### PR TITLE
Simplify the getting started guide

### PR DESCRIPTION
**Pull request by pchico83**
Fixes https://okteto.atlassian.net/browse/PROD-151

This PR does some changes in our getting started guide:

- It doesn't require changing the password for MongoDB, the password is configured by default. Setting the password has created unnecessary friction in our users. Instead, I added a note with a link to the Okteto Variables doc
- Always deploy the `seed` job to reduce the number of changes done by the user
- Reorder the dependencies and endpoints step. We got feedback that it's better to configure the endpoints after the database is configured. This way, when you access the endpoints, they work properly
- Don't set `name` in the Okteto Manifest. In the future, we could even deprecate the `name` field

The PR requires merging this [other changes](https://github.com/okteto/getting-started/pull/6). Once the PR is approved, I will apply the same changes to previous versions.

@rlamana we will also need to update the images for the Okteto UI

### CODE REVIEW

**Comment by rlamana** [comment]
@pchico83 I just updated the screenshots as discussed offline and based on the changes.

File: src/content/get-started/deploy-your-app/dependencies.mdx [hunk 1]
```diff
@@ -57,74 +66,14 @@ You can see that MongoDB is now running from the Okteto UI:
...
-
-```yaml title="okteto.yaml"
-deploy:
-  - name: Helm Install
-    command: |
-      helm upgrade --install movies chart \
-        --set mongodb.password=${MONGODB_PASSWORD} \
-        --set seed.enabled=true
-```
+And the `seed` job now succedeed to intialize the MongoDB data.
```
Comment (codyjlandstrom) [inline src/content/get-started/deploy-your-app/dependencies.mdx:69]:
```suggestion
And the `seed` job has now successfully initialized the MongoDB data.
```

File: src/content/get-started/deploy-your-app/deploy.mdx [hunk 2]
```diff
@@ -80,26 +78,51 @@ kubectl get all
...
+
+NAME             COMPLETIONS   DURATION   AGE
+job.batch/seed   0/1           2m13s      2m13s```
 
 ## Next Steps
 
 Awesome, you have deployed the Movies app to Okteto 🚀
 
-But the Okteto CLI is informing us that `There are no available endpoints for 'movies'`...
+Looks like there are still an issue with the Movies app.
```
Comment (codyjlandstrom) [inline src/content/get-started/deploy-your-app/deploy.mdx:105]:
```suggestion
Looks like there is still an issue with the Movies app.
```

File: src/content/get-started/deploy-your-app/deploy.mdx [hunk 3]
```diff
@@ -80,26 +78,51 @@ kubectl get all
...
+The `seed` job keep processing, and if you check the logs of the **api** Kubernetes Deployment:
```
Comment (codyjlandstrom) [inline src/content/get-started/deploy-your-app/deploy.mdx:106]:
```suggestion
The `seed` job keeps processing, and if you check the logs of the **api** Kubernetes Deployment:
```

**Pull request merged**